recipes/lapack/build.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
+
+# bug 145
+if test `uname` = "Linux"
+then
+  sed -e 's|/CMAKE/|/cmake/|' -i CBLAS/CMakeLists.txt
+fi
+
+mkdir build
+cd build
+
+# CMAKE_INSTALL_LIBDIR="lib" suppresses CentOS default of lib64 (conda expects lib)
+
+cmake \
+  -DCMAKE_INSTALL_PREFIX=${PREFIX} \
+  -DCMAKE_INSTALL_LIBDIR="lib" \
+  -DBUILD_TESTING=OFF \
+  -DBUILD_SHARED_LIBS=ON \
+  -DLAPACKE=ON \
+  -DCBLAS=ON \
+  ..
+
+make
```

Comment:
Yeah, sometimes you need to set fallback linking libraries. It feels a little dirty, but that is the way to get this done as `conda-build` hasn't changed everything to `@rpath`s yet. Take a look at `vigra`'s build [script]( https://github.com/conda-forge/vigra-feedstock/blob/9f68e98a1332330656241fd25225a4c184eaba92/recipe/build.sh#L79 ) too for another example.